### ISSUE TITLE
Probably would have been a good idea to set up Eslint

### ISSUE THREAD

**Issue by ashkan-pm** [babak1368/MicroserviceSample#3]
https://github.com/babak1368/MicroserviceSample/blob/5682ed34757b3f06fcd66ee35dbf763dd01edf0f/saleService/controller/customerController.js#L7

Why `let`? I mean these are small issues that can even be solved using static analysis of the code and they are in other places in the repo as well. I won't comment on all of them, but Eslint would have been able to catch these little mistakes and even fix them automatically.

Another example:
https://github.com/babak1368/MicroserviceSample/blob/5682ed34757b3f06fcd66ee35dbf763dd01edf0f/saleService/controller/customerController.js#L5

Would be nicer to have `_` for the unused variable.

**Comment by babak1368** [comment]
You are right. I am going to correct all them, however, about the ESlint, I thought about but to be honest, there were lots of works to do and finally I forgot ESlint and pushed the code without it.
I force myself to have such Static Code Analyzer even in the CI/CD pipeline and I would do it ASAP.

**Comment by babak1368** [comment]
Dear @ashkan-pm , I added and ran ESlint in the project. I also installed prettier plugin in ESlint too. All codes are formatted.

**Comment by babak1368** [comment]
Not to mention, As you know, ESlint has lots of rules that I have defined some of them that mostly due to coding style, formatting, alignment. however, I am going to define more rules.
